core/trino-parser/src/test/java/io/trino/sql/parser/TestSqlParser.java

```diff
@@ -2067,6 +2068,26 @@ public void testDropColumn()
         assertStatement("ALTER TABLE IF EXISTS foo.t DROP COLUMN IF EXISTS c", new DropColumn(QualifiedName.of("foo", "t"), identifier("c"), true, true));
     }
 
+    @Test
+    public void testAlterColumnSetDataType()
+    {
+        assertStatement("ALTER TABLE foo.t ALTER COLUMN a SET DATA TYPE bigint",
```

Comment:
`assertStatement` is deprecated (although my IDE doesn't highlight that, perhaps because this is same class)

use `assertThat(statement(....))`.